rhesus-monkeys/scripts/convert-meta.py

```diff
@@ -0,0 +1,97 @@
+import polars as pl
+import re
+from pathlib import Path
+
+# Set the folder path for raw-data
+resource_dir = Path(__file__).resolve().parent.parent
+folder_path = resource_dir / "data-raw"
+
+# Create the infant metadata files
+    # FIO There is one extra infant in 07, all infants in 08 are included in 07
+df07 = pl.read_csv(folder_path / "data07.csv")
+df08 = pl.read_csv(folder_path / "data08.csv")
+
+df_infant = df07.select(pl.col("Infant_ID"), pl.col("Mother_ID"), pl.col("GD_Delivery").alias("gestation_day_delivery")
+, pl.col("Fostered"), pl.col("Foster_ID"))
+# infant gender is in data_adult at present
+df_infant1 = df_infant.unique()
+df_infant1.write_csv("../data-raw/data_infant_meta.csv", separator=";")
```

Comment:
```suggestion
df_infant1.write_csv(folder_path / "data_infant_meta.csv", separator=";")
```

Make use of the folder path variable.